README.md

```diff
@@ -1,3 +1,35 @@
+[![Build Status](https://travis-ci.com/mstrivens/bowling-challenge.svg?branch=master)](https://travis-ci.com/mstrivens/bowling-challenge)
+
+## How to use
+
+- Fork this repo
+- Clone to repository
+
+## Testing
+
+- open SpecRunner.html
+
+## Notes
+
+- Tests are all passing, including a feature test for running a short game not including the 10th frame.
+
+- TravisCI didn't set up properly so it shows as tests failing.
+
+- The bonus system is working for retrospectively awarding scores for strikes and spares.
+
+- Extensive notes and approach in notes.md
+
+- I followed a strict TDD approach, writing tests, passing them with the code and committing.
+
+- If I had more time I would have finished the Frame Ten code. At current it is only partially functional.
+
+- The code could probably be more DRY, I felt quite unfamiliar with Javascript and as a result I wrote the methods quite longhand.
+
+- I tried to refactor using helper methods but I kept breaking the code.
+
+- I would say I am about half a day coding away from finishing the logic
+
+- I would then move onto building a app interface using HTML/CSS and JQuery.
```

Comment:
Very clear and informative README, helpful to follow your journey through the project and where you want to go